ui/app/product/directives/pnc-label-link/pnc-label-link.html

```diff
@@ -0,0 +1,48 @@
+<!--
+
+    JBoss, Home of Professional Open Source.
+    Copyright 2014 Red Hat, Inc., and individual contributors
+    as indicated by the @author tags.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<script type="text/ng-template" id="tooltipTemplateMilestone.html">
+  <strong>Start Date: </strong>{{$ctrl.item.startingDate | date:'mediumDate'}}<br>
```

Comment:
Could some of these expressions use one time binding to minimise performance impact? I can't see them changing after the page loads. see https://docs.angularjs.org/guide/expression (scroll down to One-time binding).

Just to add I realise we're not doing this elsewhere in the UI, mostly because it didn't exist when we started the project (it came in in 1.3). I'd like to start doing this where possible with new code and refactors though. This is kind of premature optimisation since we don't have any problems caused by excessive watchers, but there's also no point in setting excessive watches when we know the values won't change.